tree/ntuple/v7/test/ntuple_descriptor.cxx

```diff
@@ -123,3 +123,99 @@ TEST(RNTuple, Descriptor)
    delete[] footerBuffer;
    delete[] headerBuffer;
 }
+
+TEST(RFieldDescriptorRange, IterateOverFieldNames)
+{
+   auto model = RNTupleModel::Create();
+   auto floats = model->MakeField<std::vector<float>>("jets");
+   auto floats2 = model->MakeField<std::vector<float>>("jets");
```

Comment:
I'm surprised that this works.  I think it shouldn't because the name "jets" is already taken.

Reply:
I definitely agree, I'll check and see what we can do to prevent this (tracking issue here: https://sft.its.cern.ch/jira/browse/ROOT-10856)

Reply:
PR for this issue is: #5879